src/components/10-atoms/input-text/README.md

```diff
@@ -164,7 +164,7 @@ The Boolean attribute autofocus, when true, sets keyboard focus on the underlyin
 
 You can set the `currency` attribute to a valid [ISO 4217 currency code](https://en.wikipedia.org/wiki/ISO_4217) string. When set, &lt;input-text&gt; formats the input whenever the `onBlur` event of its internal input element fires, and also upon setting `value` programmatically. Note that the locale "style" remains fixed to `de-CH`.
 
-Currency formatting is only active when attribute `type` is set to `text`. If the inputted text can not be formatted the attribute `invalid` switches to `true`.
+Currency formatting is only active when attribute `type` is set to `text`. If the inputted text can not be formatted the attribute `invalid` switches to `true`. A valid input to format is for example `12.20`, invalid is `12.20.1` because of the two dots.
```

Comment:
would write that letters only are also invalid. i wouldn't write that 12.20 is valid because this makes the impression that only this format is allowed. Better just write what isn't allowed (to dots and only letters). there are too many allowed input formats.